api/storage_service.cc

```diff
@@ -505,6 +507,12 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
     ss::get_scylla_release_version.set(r, [](const_req req) {
         return scylla_version();
     });
+
+    ctx.db.local().get_builtin_routine_registry().register_routine("scylla_version",
+            db::make_single_value_routine<sstring>("scylla_version", [](db::builtin_routine::parameter_map&& par) {
+        return make_ready_future<sstring>(scylla_version());
+    }));
+
```

Comment:
Idea (not for now): introduce a single-line adapter that exposes a routine over the API. The routines are self-describing and so don't need additional metadata.